anthos-onprem-terraform/avmw_user_cluster_metallb/main.tf

```diff
@@ -47,12 +47,26 @@ resource "google_project_service" "default" {
   disable_on_destroy = false
 }
 
+# Update platform controller with gcloud command.
```

Comment:
Can we add more descriptive comments here?
(suggestion)
```
# This module is used to upgrade the platform controller on your admin cluster. This
# is a necessary step for the user cluster version update. If the admin cluster is 
# already on the correct version, then this module does not change anything
```

Reply:
I have used "upgrade" here to be consistent with the title. Please change it if `update` is more suitable

Reply:
Thanks for the suggestion. Added the suggested comments. I think it is more suitable to use `update` because it is also valid to update platform controller to an older version.